.gitlab-ci.yml

```diff
@@ -126,7 +79,7 @@ test_all:on-schedule:
     - export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
     - export PATH=/usr/local/cuda/bin:/opt/mpich3.2.11/bin:/opt/Xilinx/SDx/2018.2/bin:$PATH
     - export CUDA_ROOT=/usr/local/cuda
-    - NOSTATUSBAR=1 xvfb-run ./test_all.sh
```

Comment:
This is because diode1 needed an X-server, and we don't need this anymore?

Reply:
Yes, exactly :)